spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+RSpec.describe 'Post index ' do
+  before :each do
+    @user = User.create(name: 'Ricardo', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        post_counter: 0)
+    @post1 = Post.create(author_id: @user.id, title: 'Hello', text: 'first post', like_counter: 0,
+                         comment_counter: 0)
+    @post2 = Post.create(author_id: @user.id, title: 'World', text: 'second post', like_counter: 0,
+                         comment_counter: 0)
+    @post3 = Post.create(author_id: @user.id, title: '!', text: 'third post', like_counter: 0,
+                         comment_counter: 0)
+    @comment = Comment.create(user_id: @user.id, post_id: @post1.id, text: 'first comment')
+    visit user_posts_path(@user.id, page: 2)
+  end
+  it 'see the user profile picture' do
+    expect(page).to have_css("img[src*='https://unsplash.com/photos/F_-0BxGuVvo']")
+  end
+  it ' can see the user name' do
+    expect(page).to have_content('Ricardo')
+  end
+  it 'can see the number of posts user has written' do
+    expect(page).to have_content('Number of posts: 3')
+  end
+  it 'can see a post title' do
+    expect(page).to have_content('Hello')
+  end
```

Comment:
- [ ] 3 of your 24 specs test failed, including this one. Kindly fix them.

  ![image](https://github.com/RicardoEgas/blog-app/assets/81468675/dc5196c4-efb3-48cc-bb4f-6f8ba95a9d8e)